wordpress-mobile/WordPress-iOS
# Preventing emoval of Empty Paragraphs

### Details:
In this PR we're allowing multiple (empty) paragraphs in the I/O chain.

Fixes #7920

### To test:
1. Edit an existing post in Aztec
2. Type `MOO` + **newline x2** + `MOO`
3. Hit **Update**

Verify that the separation between the first and second **MOO** (s) remain after the spinner gets removed.

Needs review: @diegoreymendez 
#Moo!
